src/Summoner/CLI.hs

```diff
@@ -41,6 +47,28 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow = \case
+        -- show list of all available GHC versions
+        GhcList -> showGhcVers
+        -- show a list of all available licenses
+        LicenseList Nothing -> showLicenses
+        -- show a specific license
+        LicenseList (Just name) ->
+            case parseLicenseName (toText name) of
+                Nothing -> do
+                    errorMessage "This wasn't a valid choice."
+                    infoMessage "Here is the list of supported licenses:"
+                    showLicenses
+                    -- get and show a license`s text
+                Just licenseName -> do
+                    fetchedLicense <- fetchLicense licenseName
+                    putTextLn $ unLicense fetchedLicense
+  where
+    showGhcVers :: IO () = mapM_ (infoMessage . T.append "➤ " . showGhcVer) (reverse universe)
+    showLicenses :: IO () = mapM_ (infoMessage . (\ x -> T.append "➤ " $ show (x :: LicenseName))) universe
```

Comment:
Actually, I see could duplication here. You write the same function twice:

```haskell
showBulletList :: (Show a) => [a] -> IO ()
```

So it's better to remove code duplication and have `showEnumList` function only. And then you can call this function like this:

```haskell
showBulletList @GhcVer (reverse universe)
showBulletList @LicenseName universe
```